packages/orbit-components/src/Table/README.md

```diff
@@ -113,15 +113,15 @@ import TableCell from "@kiwicom/orbit-components/lib/Table/TableCell";
-| :------------ | :-------------- | :------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| align         | [`enum`](#enum) | `"left"` | The align of text in the TableCell.                                                                                                                                 |
-| as            | [`enum`](#enum) | `"td"`   | possibility to render TableCell in different HTML                                                                                                                   |
-| children      | `React.Node`    |          | The content of the TableCell.                                                                                                                                       |
-| dataTest      | `string`        |          | Optional prop for testing purposes.                                                                                                                                 |
-| scope         | [`enum`](#enum) |          | The scope attribute identifies whether a table header is a column header or a row header. More about a11y reasons [here](https://webaim.org/techniques/tables/data) |
-| verticalAlign | [`enum`](#enum) |          | The vertical align of the content in the TableCell.                                                                                                                 |
-| whiteSpace    | [`enum`](#enum) |          | The white-space setting of text in the TableCell.                                                                                                                   |
+| Name          | Type            | Default  | Description                                                                                                                      |
+| :------------ | :-------------- | :------- | :------------------------------------------------------------------------------------------------------------------------------- |
+| align         | [`enum`](#enum) | `"left"` | The align of text in the TableCell.                                                                                              |
+| as            | [`enum`](#enum) | `"td"`   | Possibility to render TableCell in different HTML.                                                                               |
+| children      | `React.Node`    |          | The content of the TableCell.                                                                                                    |
+| dataTest      | `string`        |          | Optional prop for testing purposes.                                                                                              |
+| scope         | [`enum`](#enum) |          | Can be set only when `as="th"`. Identifies whether a table header is a column header or a row header. See the Accessibility tab. |
```

Comment:
Is it possible to link the accessibility tab?

Reply:
I saw that Daniel advised not to link the tab here https://github.com/kiwicom/orbit/pull/4616#discussion_r1961853428

Reply:
Okay, not a big deal.